app/views/_controls.html.erb

```diff
@@ -9,7 +9,7 @@
               <li <%= 'class=active' if request.url.include? 'dashboard' %>><a href="/dashboard/" >DASHBOARD</a></li>
               <% end %>
               <li <%= 'class=active' if current_page?(sufia.about_path) %>><a href="/about/" >ABOUT</a></li>
-              <li <%= 'class=active' if current_page?(help_path) %>><a href="/help/" >HELP</a></li>
+              <li><a href="http://guides.lib.vt.edu/" >HELP</a></li>

```

Comment:
I do not think that is necessary, as this will never need translation, and is only used in one place.